### PR TITLE
[OSDOCS-10471] :Add dedicated page for GCP Filestore CSI Driver Operator

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.15+
Issue:

https://issues.redhat.com/browse/OSDOCS-10471
Link to docs preview:

**OSD:**
https://75622--ocpdocs-pr.netlify.app/openshift-dedicated/latest/storage/container_storage_interface/persistent-storage-csi-google-cloud-file

**OCP:** (this was a change to OSD docs only, but both assemblies share the module. OCP docs have not been changed in terms of output info.)
https://75622--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-google-cloud-file.html#persistent-storage-csi-olm-operator-install_persistent-storage-csi-google-cloud-file
QE review:
- [x] QE has approved this change.

Additional information:

image

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @xueli181114, can you please review this PR when you get a moment? Thanks!

**Comment by svmrh** [comment]
Thanks @mletalie for getting this drafted in such a short duration. 

While we wait for @xueli181114 or @lento-sun to review this PR, I have some comment on this change:

1. While the prerequisites to OpenShift Dedicated web console is correct, to install the "GCP Filestore CSI Operator" from the Operator Hub, the user has to log into the OpenShift console from the OpenShift Cluster Manager (OCM) UI being referred as "OpenShift Dedicated web console".  

> Installing the GCP Filestore CSI Driver Operator
> ..
> Prerequisites
> - Access to the OpenShift Dedicated web console.
> 
> Procedure
> To install the GCP Filestore CSI Driver Operator from the web console:
> - Log in to the web console.

May be we can follow the same approach as listed in this section -
https://75622--ocpdocs-pr.netlify.app/openshift-dedicated/latest/operators/admin/olm-adding-operators-to-cluster
"Access to an OpenShift Dedicated cluster using an account with the dedicated-admin role."
"Navigate in the web console to the Operators → OperatorHub page."

> Installing from OperatorHub using the web console
> You can install and subscribe to an Operator from OperatorHub by using the OpenShift Dedicated web console.
> 
> Prerequisites
> Access to an OpenShift Dedicated cluster using an account with the dedicated-admin role.
> 
> Procedure
> Navigate in the web console to the Operators → OperatorHub page.

**Comment by mletalie** [comment]
> Thanks @mletalie for getting this drafted in such a short duration.
> 
> While we wait for @xueli181114 or @lento-sun to review this PR, I have some comment on this change:
> 
> 1. While the prerequisites to OpenShift Dedicated web console is correct, to install the "GCP Filestore CSI Operator" from the Operator Hub, the user has to log into the OpenShift console from the OpenShift Cluster Manager (OCM) UI being referred as "OpenShift Dedicated web console".
> 
> > Installing the GCP Filestore CSI Driver Operator
> > ..
> > Prerequisites
> > 
> > * Access to the OpenShift Dedicated web console.
> > 
> > Procedure
> > To install the GCP Filestore CSI Driver Operator from the web console:
> > 
> > * Log in to the web console.
> 
> May be we can follow the same approach as listed in this section - https://75622--ocpdocs-pr.netlify.app/openshift-dedicated/latest/operators/admin/olm-adding-operators-to-cluster "Access to an OpenShift Dedicated cluster using an account with the dedicated-admin role." "Navigate in the web console to the Operators → OperatorHub page."
> 
> > Installing from OperatorHub using the web console
> > You can install and subscribe to an Operator from OperatorHub by using the OpenShift Dedicated web console.
> > Prerequisites
> > Access to an OpenShift Dedicated cluster using an account with the dedicated-admin role.
> > Procedure
> > Navigate in the web console to the Operators → OperatorHub page.

Thanks @svmrh, I appreciate the quick follow up! Will read through these notes and get with you regarding questions.

**Comment by mletalie** [comment]
https://issues.redhat.com/browse/OCPSTRAT-498

**Comment by duanwei33** [comment]
It looks good to me from storage qe perspective.

**Review by mletalie — COMMENTED** [review]
/label peer-review-needed

File: modules/persistent-storage-csi-gcp-file-install.adoc [hunk 1]
```diff
@@ -6,16 +6,30 @@
...
 * Access to the {product-title} web console.
 
 .Procedure
 To install the GCP Filestore CSI Driver Operator from the web console:
 
+ifdef::openshift-dedicated[]
+
+. Log in to the {cluster-manager-url} and click the name of your cluster.
+
+. Select your cluster.
```
Comment (bhardesty) [inline modules/persistent-storage-csi-gcp-file-install.adoc:22]: The previous step says to click the name of your cluster, and this step says to select your cluster. Is there a difference between these two actions? 

If this comment makes no sense when you're actually looking at the console, feel free to disregard!

**Review by bhardesty — COMMENTED** [review]
Thanks - this looks good, and I verified that the conditionalization works as intended. I just have one small question, which may be irrelevant.

File: modules/persistent-storage-csi-gcp-file-install.adoc (hunk 1 above)
Comment (mletalie) [reply modules/persistent-storage-csi-gcp-file-install.adoc:22]: Hi @bhardesty,
   Makes total sense and thanks for catching. Seemed to make sense on Friday, not so much today! :) 
Have updated and removed the repeated step.

**Pull request merged**